cli/azd/pkg/infra/provisioning/bicep/prompt.go

```diff
@@ -103,7 +103,23 @@ func (a *BicepProvider) locationsWithQuotaFor(
 	ctx context.Context, subId string, locations []string, quotaFor []string) ([]string, error) {
 	var sharedResults sync.Map
 	var wg sync.WaitGroup
+
+	azureAiServicesLocations, err := a.azureClient.GetResourceSkuLocations(
+		ctx, subId, "AIServices", "S0", "Standard", "accounts")
```

Comment:
Can we add some comments here about why we are hard coding the SKU, tier, etc.  What happens if the user wants "GlobalStandard" instead? Are we supporting that?